Core/Helper.php

```diff
@@ -5,18 +5,31 @@
  * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
  * - License can be found under:
  * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
- *
  */
 
 namespace Wirecard\Oxid\Core;
 
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\Eshop\Core\Model\ListModel;
+use OxidEsales\Eshop\Application\Model\Payment;
+
 /**
  * Util functions
  */
 class Helper
 {
     /**
+     * Gets the translation for a given key.
      *
+     * @param string $sKey
+     * @return string
+     */
+    public static function translate(string $sKey): string
```

Comment:
This is all ok i just saw a method isWirecardPaymentMethod in this file, this is wrong think about white labeling :)